AdminApi/function.json

```diff
@@ -0,0 +1,18 @@
+{
+  "scriptFile": "index.js",
```

Comment:
this needs to reference `../lib/index.js`

Reply:
yes, I haven't merged master branch yet